src/modelfree/common/utils.py

```diff
@@ -149,39 +155,114 @@ def _reset_video_recorder(self, metadata=None):
         )
 
 
+class ReporterOutputFormat(KVWriter):
+    """Key-value logging plugin for Stable Baselines that writes to a Ray Tune StatusReporter."""
+    def __init__(self, reporter):
+        self.last_kvs = dict()
+        self.reporter = reporter
+
+    def writekvs(self, kvs):
+        self.last_kvs = kvs
+        self.reporter(**kvs)
+
+
 def make_session(graph=None):
     tf_config = tf.ConfigProto()
     tf_config.gpu_options.allow_growth = True
     sess = tf.Session(graph=graph, config=tf_config)
     return sess
 
 
-def simulate(venv, policies, render=False):
+class TrajectoryRecorder(object):
```

Comment:
Needs a docstring

Reply:
Consider making `TrajectoryRecorder` a `MultiVecWrapper`. Then it can be easily dropped into a variety of situations, not just `simulate`.